File: String/771jewelsandStones.py
class Solution(object):
    def numJewelsInStones(self, J, S):
        """
        :type J: str
        :type S: str
        :rtype: int
        """
        res = 0
        dic = {}
        for i in S:
            if i not in dic:
                dic[i] = 1
            else:
                dic[i] += 1
        for i in dic.keys():
            if i in J:
                res += dic[i]
                
        return res

File: String/771JewelsandStones.cpp
class Solution {
public:
    int numJewelsInStones(string J, string S) {
        int cnt = 0;
        for(int i=0; i<J.size(); i++){
            for(int j=0; j<S.size(); j++){
                if(J[i]==S[j]) cnt++;
            }
        }
        return cnt;
    }
};